web/src/lib/search/streamingQa.ts

```diff
@@ -152,7 +152,7 @@ export const searchRequestStreamed = async ({
           const topDocuments = chunk.top_documents as any[] | null;
           if (topDocuments) {
             relevantDocuments = topDocuments.map(
```

Comment:
nit: `chunk.top_documents as any[]` -> `chunk.top_documents as DanswerDocument[]` and then line 154 not needed